lms/djangoapps/instructor_task/api.py

```diff
@@ -68,6 +69,34 @@ def get_running_instructor_tasks(course_id):
     return instructor_tasks.order_by('-id')
 
 
+def _get_filtered_instructor_tasks(course_id, user, task_types):
+    """
+    Returns a filtered query of InstructorTask to use for listing canvas tasks
```

Comment:
This was probably moved over directly, but this docstring isn't really accurate 

```suggestion
    Returns a filtered query of InstructorTasks based on the course, user, and desired task types
```